### DISCUSSION TITLE
Regeneration Meter | RuneLITE theme pack compatibilty

### DISCUSSION THREAD

**Discussion by Varietyz** [runelite/runelite#18304] [Ideas]
Hello!
I am the designer of the RuneLITE theme pack from the Resource Packs Plugin.
I have my orbs designed in a unique lite way and would love the regeneration meter visual to comply with the orbs.

Multiple pack users have reached out to me to ask for this feature.
Myself, cannot add a horizontal regenmeter underneath the orbs.

Right now it is looking like this:
![image](https://github.com/user-attachments/assets/32da521b-fa2b-4bfe-a506-7c411e4c93af)

it be nice to have it regenerate from left to right at the bottom, just right underneath the orb's frame itself, as in this example.
![image](https://github.com/user-attachments/assets/1334b7a4-0624-4f0d-bb4a-6010bfb1b7e6)

Would there be any possibility whatsoever?

Thank you kindly <3

**Comment by 0desza** [comment]
This would be fantastic! Hope you can get the community support needed for this idea

**Comment by Varietyz** [comment]
Discussion may be closed, i’ve released a plugin to accomodate this.

Plugin: LITE Regen Meter
